### DISCUSSION TITLE
✨ WatchYourLAN

### DISCUSSION THREAD

**Discussion by OriginalSimon** [runtipi/runtipi-appstore#4637] [Requests]
### Description

 Lightweight network IP scanner. Can be used to notify about new hosts and monitor host online/offline history 

### Documentation link

https://github.com/aceberg/WatchYourLAN

### Last application release & date

 2.0.0 (31/08/2024)

### Application license

MIT license

### Source code repository link

https://github.com/aceberg/WatchYourLAN

### Docker image link

https://hub.docker.com/r/aceberg/watchyourlan

### Please confirm the following

- [X] I confirm that I've put the app name in the title of the discussion
- [X] I confirm that the application is actively maintained.
- [X] I confirm that the docker image of the application is actively maintained.
- [X] I confirm that I have indeed checked that no similar request for this application already exists

### Gathering crowds

Hey folks!

Please upvote ⬆️ this discussion to show your interest in this request! 

Thanks ⛺

**Comment by ChimairaSpawn** [comment]
Good suggestion, it looks like it has been added since you submitted this request!
